### PR TITLE
An env variable can set Canoe's language.

### PR DESCRIPTION
**Pull request by sandroroux**
Closes catalpainternational/hamahon#250.

### CODE REVIEW

**Review by katevoss — DISMISSED** [review]

File: src/js/redux/Store.js [hunk 1]
```diff
@@ -4,17 +4,30 @@ import { reducers } from "./Reducers";
...
+    }
+
+    // Allow an env-set default language.
+    if ([TETUN_LANG_CODE, ENGLISH_LANG_CODE].includes(envLanguage)) {
+        localStorage.setItem(LANGUAGE_STORAGE_KEY, envLanguage);
+        return envLanguage;
+    }
+
+    // Set language with the browser setting.
+    if (navigator.language.includes(ENGLISH_LANG_CODE)) {
```
Comment (PeteCoward) [inline src/js/redux/Store.js:25]: does not seem like we need an if here, am I being blind, both if branches look the same

Comment (sandroroux) [reply src/js/redux/Store.js:25]: Good catch. It's a typo. I'll fix it.

File: src/js/redux/Store.js [hunk 2]
```diff
@@ -4,17 +4,30 @@ import { reducers } from "./Reducers";
...
+        localStorage.setItem(LANGUAGE_STORAGE_KEY, ENGLISH_LANG_CODE);
+        return ENGLISH_LANG_CODE;
     } else {
-        localStorage.setItem(LANGUAGE_STORAGE_KEY, "tet");
-        return "tet";
+        localStorage.setItem(LANGUAGE_STORAGE_KEY, ENGLISH_LANG_CODE);
```
Comment (sandroroux) [inline src/js/redux/Store.js:29]:
```suggestion
        localStorage.setItem(LANGUAGE_STORAGE_KEY, TETUN_LANG_CODE);
```

File: src/js/redux/Store.js [hunk 3]
```diff
@@ -4,17 +4,30 @@ import { reducers } from "./Reducers";
...
+        return ENGLISH_LANG_CODE;
```
Comment (sandroroux) [inline src/js/redux/Store.js:30]:
```suggestion
        return TETUN_LANG_CODE;
```

**Review by PeteCoward — APPROVED** [review]

**Pull request merged**